### PR TITLE
Refactored bin commands into separate files

### PR DESCRIPTION
**Pull request by syrusakbary**
# Description

Refactored bin commands into separate files.
This PR does not do any sustancial changes other than refactoring into different files for better readability.

# Review

- [x] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

File: src/commands/run.rs [hunk 1]
```diff
@@ -0,0 +1,982 @@
+use crate::common::{get_cache_dir, PrestandardFeatures};
+use crate::utils::read_file_contents;
+use std::collections::HashMap;
+use std::fs::read_to_string;
+#[cfg(feature = "backend-llvm")]
+use std::fs::File;
+use std::path::PathBuf;
+use std::process::exit;
+use std::str::FromStr;
+use wasmer_runtime::compile_with_config_with;
```
Comment (nlewycky) [inline src/commands/run.rs:10]: Move this down with rest of `wasmer_runtime`?

File: src/utils.rs [hunk 2]
```diff
@@ -120,6 +124,16 @@ pub fn parse_args(
     }
 }
 
+/// Read the contents of a file
+pub fn read_file_contents(path: &PathBuf) -> Result<Vec<u8>, io::Error> {
+    let mut buffer: Vec<u8> = Vec::new();
+    let mut file = File::open(path)?;
+    file.read_to_end(&mut buffer)?;
+    // We force to close the file
+    drop(file);
```
Comment (MarkMcCaskey) [inline src/utils.rs:133]: This doesn't do anything, we can remove it!

**Comment by Hywan** [comment]
Rage against the machine 🤘.

**Pull request merged**